handlers/lobbies.js

```diff
@@ -30,11 +31,125 @@ Dota2._lobbyOptions = {
 };
 
 // Methods
-Dota2.Dota2Client.prototype.createPracticeLobby = function(password, options, callback) {
+
+/**
+ * Sends a message to the Game Coordinator requesting to create a lobby. Listen for
+ * practiceLobbyUpdate response for a snapshot-update of the newly created lobby.
+ *
+ * @param {object} options
+ * @param {string} options.game_name
+ * @param {string} options.pass_key
+ * @param {ServerRegion} [options.server_region]
+ * @param {DOTA_GameMode} [options.game_mode=DOTA_GAMEMODE_AP]
+ * @param {DOTAGameVersion} [options.game_version=GAME_VERSION_STABLE]
+ * @param {DOTA_CM_PICK} [options.cm_pick=DOTA_CM_RANDOM]
+ * @param {boolean} [options.allow_cheats=false]
+ * @param {boolean} [options.fill_with_bots=false]
+ * @param {boolean} [options.allow_spectating=true]
+ * @param {SeriesType} [options.series_type=NONE]
+ *
+ * Whether or not the game is part of a series (Bo3, Bo5).
+ *
+ * @param {number} [options.radiant_series_wins=0]
+ *
+ * # of games won so far, e.g. for a Bo3 or Bo5.
+ *
+ * @param {number} [options.dire_series_wins=0]
+ *
+ * # of games won so far, e.g. for a Bo3 or Bo5.
+ *
+ * @param {number} [options.previous_match_override]
+ *
+ * In a series, the match ID of the previous game. If not supplied, the GC will try
```

Comment:
You don't need to add the `@class Dota2.Dota2Client`, I'm preparing a pull where I'm cleaning up index.js to make it compatible with the CommonJS syntax for JSDoc. As soon as that one's pulled this will sort itself out.